migrator/migrator.go

```diff
@@ -493,9 +493,11 @@ func (m Migrator) GuessConstraintAndTable(stmt *gorm.Statement, name string) (_
 	}
 
 	if field := stmt.Schema.LookUpField(name); field != nil {
-		for _, cc := range checkConstraints {
-			if cc.Field == field {
-				return nil, &cc, stmt.Table
+		// fix return value for *schema.Check
```

Comment:
Remove this comment, this message is for PR, not for code.

Reply:
@huacnlee i has reverted this line.